From 3f90531563abcdf92b06eaed93d9880a4fc4d718 Mon Sep 17 00:00:00 2001
From: Chih-Wei Huang <cwhuang@linux.org.tw>
Date: Tue, 18 Oct 2016 23:35:04 +0800
Subject: [PATCH 21/28] Ignore screen size when determines
 mForceDefaultOrientation

Conflicts:
	services/core/java/com/android/server/policy/PhoneWindowManager.java

Change-Id: Ic0ef774c745116a392aab7c125aed508076aa3a5
---
 .../core/java/com/android/server/policy/PhoneWindowManager.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/java/com/android/server/policy/PhoneWindowManager.java b/services/core/java/com/android/server/policy/PhoneWindowManager.java
index 23ee87851e14..dcb9e6e55930 100644
--- a/services/core/java/com/android/server/policy/PhoneWindowManager.java
+++ b/services/core/java/com/android/server/policy/PhoneWindowManager.java
@@ -2488,7 +2488,7 @@ public class PhoneWindowManager implements WindowManagerPolicy {
         // so if the orientation is forced, we need to respect that no matter what.
         final boolean isTv = mContext.getPackageManager().hasSystemFeature(
                 PackageManager.FEATURE_LEANBACK);
-        mForceDefaultOrientation = ((longSizeDp >= 960 && shortSizeDp >= 720) || isCar || isTv) &&
+        mForceDefaultOrientation = /* longSizeDp >= 960 && shortSizeDp >= 720 && */
                 res.getBoolean(com.android.internal.R.bool.config_forceDefaultOrientation) &&
                 // For debug purposes the next line turns this feature off with:
                 // $ adb shell setprop config.override_forced_orient true
-- 
2.17.1

